docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -366,6 +366,12 @@ it can use the domain name from the server hostname as a proxy for the zone.
 If there is no other source of zone data, then a guess is made, based on the client configuration (as opposed to the instance configuration).
 We take `eureka.client.availabilityZones`, which is a map from region name to a list of zones, and pull out the first zone for the instance's own region (that is, the `eureka.client.region`, which defaults to "us-east-1", for compatibility with native Netflix).
 
+=== AOT and Native Image Support
+
+Spring Cloud Netflix Eureka Client integration supports Spring AOT transformations and native images, however, only with refresh mode enabled.
```

Comment:
I'm a bit confused by the sentence. It seems to contradict the warning below regarding refresh mode.

Reply:
You're right - it was late. I wanted to write "disabled". Fixing now.